tattoo_shop/demo/website_views.xml

```diff
@@ -242,42 +242,20 @@
                                 </label>
                                 <div class="col-sm">
                                   <div class="row s_col_no_resize s_col_no_bgcolor s_website_form_multiple" data-name="tag_ids" data-display="horizontal">
-                                    <div class="checkbox col-12 col-lg-4 col-md-6">
-                                      <div class="form-check">
-                                        <input type="checkbox" class="s_website_form_input form-check-input" id="od2wplpr6v4a0" name="tag_ids" t-att-value="request.env.ref('tattoo_shop.crm_tag_1').id" data-fill-with="undefined"/>
-                                        <label class="form-check-label s_website_form_check_label" for="od2wplpr6v4a0">Tattoo</label>
+                                    <t t-set="tag_values" t-value="request.env['crm.tag'].sudo().search([])"/>
```

Comment:
This needs to be validated and requires further brainstorming, because it feels wrong. CRM tags are mostly backend tools, and we want to expose only some of them.
We won't apply this strategy here (you can remove the commit)